Source/items.h

```diff
@@ -348,6 +349,20 @@ struct ItemStruct {
 			return false;
 		}
 	}
+
+	UiFlags getTextColor() const

```

Comment:
Aren't we using PascalCase for method names?

Reply:
`ClassMethodCase` is not defined in `.clang-tidy`.
There is `isJewelry` and `isHelm` in the same class, so I followed that.

Reply:
Can we look into this @AJenbo ? Again, the earlier we decide on the format, the less changes there will be.